src/frontc/cparser.mly

```diff
@@ -1693,5 +1717,16 @@ asmclobberlst_ne:
    one_string_constant                           { [$1] }
 |  one_string_constant COMMA asmclobberlst_ne    { $1 :: $3 }
 ;
+asmlabels:
+    /* empty */                         { None }
+| COLON asmlabelslst                    { $2 } /* This is not asmlabelslst_ne because we distinguish between no labels field (None) and no labels provided (Some []) */
+;
```

Comment:
Is just having the colon on its own and then no labels actually legal code?

Reply:
No, I originally thought it was. Now the distinction between Some [] and None is not necessary anymore.